src/c++/perf_analyzer/concurrency_worker.cc

```diff
@@ -197,8 +196,6 @@ ConcurrencyWorker::AsyncCallbackFinalize(uint32_t ctx_id)
     notified_ = true;
   }
 
-  total_ongoing_requests_--;
```

Comment:
This can be removed from Finalize now and to a common location. Before, this line very explicitly had to be after the lines above and before the lines below.